src/dsm/utility/DijkstraResult.hpp

```diff
@@ -0,0 +1,38 @@
+/// @file utility/DijkstraResult.hpp
+/// @brief This file contains the definition of the DijkstraResult class.
+///
+/// @details The DijkstraResult class represents the result of a Dijkstra algorithm.
+/// 		 It is templated by the type of the graph's id.
+
+#ifndef DijkstraResult_hpp
+#define DijkstraResult_hpp
+
+namespace dsm {
+
+  /// @brief The DijkstraResult class represents the result of a Dijkstra algorithm.
+  /// @tparam Id, The type of the graph's id. Must be an unsigned integral type.
+  template <typename Id>
+    requires std::unsigned_integral<Id>
+  class DijkstraResult {
```

Comment:
Does this class handle multiple paths with equal length?

Reply:
Why not?

Reply:
It returns only a vector, doesn't it?